lib/service/service.go

```diff
@@ -1084,10 +1084,17 @@ func initUploadHandler(ctx context.Context, auditConfig types.ClusterAuditConfig
 		}
 		return handler, nil
 	case teleport.SchemeS3:
-		config := s3sessions.Config{}
+		var fips *bool
+		if useFIPS {
+			fips = new(bool)
+			*fips = true
+		}
+		config := s3sessions.Config{UseFIPSEndpoint: fips}
```

Comment:
AWS SDK provides methods for converting scalar values to and from pointers (e.g. [Bool()](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#Bool)) - can they be used to make this code block less "awkward"? E.g.

```suggestion
		config := s3sessions.Config{UseFIPSEndpoint: aws.Bool(useFIPS)}
```